pyomo/contrib/doe/doe.py

```diff
@@ -2323,3 +2358,95 @@ def _sgn(self, p):
             return 1
         else:
             return -1
+
+
+# loggers for the functions
+function_logger = logging.getLogger(__name__)
+function_logger.setLevel(logging.WARNING)
```

Comment:
The logger for this module is retrieved above, with the logging level set based on the user input. See:
https://github.com/Pyomo/pyomo/blob/9efc82d3e18777a2c725586c036e209f6cbb79e9/pyomo/contrib/doe/doe.py#L218-L220
I'm worried that grabbing the logger for this module again and setting the level here will lead to weird edge cases with the logging code in the class. What's the reasoning for manually setting the level here?

@adowling2 would you be opposed to moving these stand-alone functions into `pyomo.contrib.doe.utils`?

Reply:
If you move these functions into `pyomo.contrib.doe.utils`, then I would also suggest combining the test files in this PR into a single `test_utils.py` file.